test/e2e/framework/expect.go

```diff
@@ -43,6 +46,18 @@ func ExpectNoError(err error, explain ...interface{}) {
 // ExpectNoErrorWithOffset checks if "err" is set, and if so, fails assertion while logging the error at "offset" levels above its caller
 // (for example, for call chain f -> g -> ExpectNoErrorWithOffset(1, ...) error would be logged for "f").
 func ExpectNoErrorWithOffset(offset int, err error, explain ...interface{}) {
+	if err != nil {
+		Logf("Unexpected error:\n%s", format.Object(err, 1))
+		prefix := ""
+		if len(explain) > 0 {
+			if str, ok := explain[0].(string); ok {
+				prefix = fmt.Sprintf(str, explain[1:]...) + ": "
+			} else {
+				prefix = fmt.Sprintf("unexpected explain arguments, need format string: %v", explain)
+			}
+		}
+		Fail(prefix+err.Error(), 1+offset)
+	}
 	gomega.ExpectWithOffset(1+offset, err).NotTo(gomega.HaveOccurred(), explain...)
```

Comment:
That's a leftover from the version where gomega was called for other, non-nil errors. Now that all non-nil errors are handled by our own code it can be removed.

Reply:
Done.